mitosheet-helpers/mitosheet_helper_enterprise/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mitosheet_helper_enterprise"
+version = "0.0.3"
```

Comment:
This version does not match the version listed in `mitosheet-helpers/mitosheet_helper_enterprise/setup.cfg`